docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -130,43 +152,39 @@ occupation STRING,
 zipcode STRING)
 ROW FORMAT DELIMITED
 FIELDS TERMINATED BY '|'
-STORED AS TEXTFILE;
-
-hive> LOAD DATA LOCAL INPATH '<path_to_ml-100k>/u.user'
-OVERWRITE INTO TABLE u_user;
+LOCATION 'alluxio://master_hostname:port/ml-100k';
 ```
 
 View Alluxio WebUI at `http://master_hostname:19999` and you can see the directory and file Hive creates:
 
 ![HiveTableInAlluxio]({{site.data.img.screenshot_presto_table_in_alluxio}})
 
-Alternatively, you can follow the [instructions](Running-Hive-with-Alluxio.html#create-new-tables-from-files-in-alluxio) to create the tables from existing files in Alluxio.
+### Start Hive metastore
 
-Next, start your Hive metastore service. Hive metastore listens on port 9083 by default.
+Next, ensure your Hive metastore service is running. Hive metastore listens on port `9083` by
+default. If it is not running,
 
 ```bash
-$ /<PATH_TO_HIVE>/bin/hive --service metastore
+$ ${HIVE_HOME}/bin/hive --service metastore
 ```
 
-The following is an example of the Presto configuration `/<PATH_TO_PRESTO>/etc/catalog/hive.properties` :
-
-```properties
-connector.name=hive-hadoop2
-hive.metastore.uri=thrift://localhost:9083
-hive.config.resources=/<PATH_TO_HADOOP>/etc/hadoop/core-site.xml,/<PATH_TO_HADOOP>/etc/hadoop/hdfs-site.xml
-```
+### Start Presto server
 
-Start your Presto server. Presto server runs on port 8080 by default:
+Start your Presto server. Presto server runs on port `8080` by default (set by 
+`http-server.http.port` in `${PRESTO_HOME}/etc/config.properties` ):
 
 ```bash
-$ /<PATH_TO_PRESTO>/bin/launcher run
+$ ${PRESTO_HOME}/bin/launcher run
 ```
 
+### Query tables using Presto
+
 Follow [Presto CLI guidence](https://prestodb.io/docs/current/installation/cli.html) to download the `presto-cli-<PRESTO_VERSION>-executable.jar`,
-rename it to `presto`, and make it executable with `chmod +x` 
-(sometimes the executable `presto` exists in `/<PATH_TO_PRESTO>/bin/presto` and you can use it directly).
+rename it to `presto`, and make it executable with `chmod +x`
+(sometimes the executable `presto` exists in `${PRESTO_HOME}/bin/presto` and you can use it
+directly).
 
-Run a single query similar to:
+Run a single query similar to (assume the server is running locally at `8080`):
```

Comment:
This reads strange with the parenthesis between "similar to" and the example.

How about this?

Run a single query (replace `localhost:8080` with your presto server hostname and port)

Reply:
fixed